templates/head-meta.html

```diff
@@ -10,7 +10,7 @@
 
 <link rel="publisher" href="https://plus.google.com/107187849809354688692">
 <link rel="shortcut icon" href="/images/logos/p-logo-32.png">
-<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Source+Code+Pro:400,700|Roboto+Slab:400,700">
+<link rel="preload" href="//fonts.googleapis.com/css?family=Source+Code+Pro:400,700|Roboto+Slab:400,700">
```

Comment:
I thought `link[rel=preload]` doesn't actually include the resource in the page; it only initiates the fetch (yes, it shows up in dev tools timeline, but that's only the preload). Maybe test this in a browser that doesn't have Slabs/Source Code Pro and see if it's the expected result?